api/projects/tests/test_permissions.py

```diff
@@ -371,3 +372,29 @@ def test_regular_user_has_no_destroy_permission(self):
 
         # Then - exception thrown
         assert not result
+
+
+@pytest.mark.django_db
+def test_free_plan_has_only_fixed_projects_permission():
```

Comment:
Can we try to follow the `# Given / When / Then` pattern here. The test is structured quite nicely, but I'd love to see the sections split out appropriately with comments.